### PR TITLE
Pip installation is not working

### PR DESCRIPTION
**Pull request by hmunfru**
@jesuspg @flopezag 
Installation package defined in setup.py is wrong. Installation is not working. This PR resolves it

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
```
Comment (flopezag) [inline setup.py:34]: **version** should be a parameter in the settings file.

File: setup.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 
 setup(
 name='fiware-glancesync',
-packages=['fiware-glancesync'],  # this must be the same as the name above
+packages=['glancesync'],  # this must be the same as the name above
```
Comment (flopezag) [inline setup.py]: I prefer to specify which one no and not which yes.

I mean:

  packages=find_packages(exclude=['_tests_']),

**Comment by flopezag** [comment]
missing setup.cfg.

In the same line, I should include the bash script file to start/stop/restart the service when you install it

**Comment by flopezag** [comment]
the name of the branch is not correct and missing correct description of the PR.

**Comment by flopezag** [comment]
Just other comment, if we want to use supervisor, we should add the configuration files in /utils (for example).

**Comment by hmunfru** [comment]
But this is not part of this task right? I should include a new one for it

**Comment by hmunfru** [comment]
setup.cfg is in https://github.com/telefonicaid/fiware-glancesync/blob/develop/setup.cfg

File: README.rst [hunk 3]
```diff
@@ -477,6 +477,13 @@ This configuration file defines two *targets*: ``master`` and ``experimental``.
 synchronises all the public images with properties *nid* and/or *type* defined. The last one only
 synchronises images with ``type=baseimages``
 
+Installation with Docker
+------------------------
+By using the Dockerfile, it is possible to generate the image
+  docker build -t fiware-glancesync .
```
Comment (flopezag) [inline README.rst]: Try to leave a white space after and before the command line and put the following content:

.. code::
    docker build -t fiware-glancesync .

and

..code::
    docker run -ti -e OS_USERNAME={OS_USERNAME} -e OS_PASSWORD={OS_PASSWORD} -e OS_AUTH_URL={OS_AUTH_URL} -e OS_TENANT_NAME={OS_TENANT_NAME} -e OS_REGION={OS_REGION} fiware-glancesync

File: glancesync/glancesync_config.py [hunk 4]
```diff
@@ -23,6 +23,7 @@
 # contact with opensource@tid.es
 #
 __author__ = 'chema'
+__version__ = '1.2.0.6'
```
Comment (flopezag) [inline glancesync/glancesync_config.py]: Why 1.2.0.6?

If you move the version to this file you have to reconfigure the .bumpversion.cfg file.

Besides, I think that glancesync_config.py and glancesync.conf should be included in settings (glancesync_settings).

File: glancesync/glancesync_settings/jenkins.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2014 Telefónica Investigación y Desarrollo, S.A.U
+#
```
Comment (flopezag) [inline glancesync/glancesync_settings/jenkins.py]: Why you put this file and why inside glancesync_settings?

I suggest to use settings and not glancesync_settings.

**Comment by flopezag** [comment]
The description of the PR is not following the defined style guide and the name of the PR should have been:

feature/5726-Pip-installation-not-working

or

bug/5726-Pip-installation-not-working

**Comment by flopezag** [comment]
You have conflicts with the last release I assume.

File: setup.py (hunk 1 above)
Comment (hmunfru) [inline setup.py:34]: done

**Comment by hmunfru** [comment]
Conflics have been solved. 
I cannot change the brach name if I want to keep this PR

**Comment by flopezag** [comment]
LGTM

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**